content/en/guides/get-started/routing.md

```diff
@@ -36,20 +42,20 @@ The `<NuxtLink>` component should be used for all internal links. That means for
 <template>
   <main>
     <h1>Home page</h1>
-    <NuxtLink to="/about"
-      >About (internal link that belongs to the Nuxt App)</NuxtLink
-    >
+    <NuxtLink to="/about">
+      About (internal link that belongs to the Nuxt App)
+    </NuxtLink>
     <a href="https://nuxtjs.org">External Link to another page</a>
   </main>
 </template>
 ```
 
 <app-modal>
-  <code-sandbox  :src="csb_link"></code-sandbox>
+  <code-sandbox :src="csb_link" />
```

Comment:
You cannot use self-closing tags, i.e., this won't work in content module
https://content.nuxtjs.org/writing#vue-components

Reply:
Got it, fixed - 4bd375a